arangod/Transaction/ClusterUtils.cpp

```diff
@@ -35,7 +35,23 @@ namespace arangodb {
 namespace transaction {
 namespace cluster {
 
+void abortTransactions(LogicalCollection& coll) {
+  transaction::Manager* mgr = transaction::ManagerFeature::manager();
+  if (!mgr) { // might be called during shutdown
```

Comment:
This looks very much like a race condition. Suppose this is called during shutdown, but _before_ `ManagerFeature::unprepare`. Then we would get a valid `ptr` to an object that can be destroyed at any time.

Reply:
not gonna happen, comment is wrong

Reply:
only nullptr during unit testing